From c9ea705fcdc7e7a650393902dd0ef425cc4913ee Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Thu, 4 Jan 2018 14:43:45 +0200
Subject: [PATCH 3192/5242] MLK-17275-13: arm64: dts: fsl-imx8qxp-mek: Remove
 dsi/lvds specific dts files

commit  5661dd265ce938afed8c7178f2e57d357274f716 from
https://source.codeaurora.org/external/imx/linux-imx.git

Delete the dst files specific to dsi/lvds nodes for the 8QXP MEK platform.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/Makefile             |    4 -
 .../dts/freescale/fsl-imx8qxp-mek-dsi-adv7535.dts  |   95 --------------------
 .../dts/freescale/fsl-imx8qxp-mek-lvds0-it6263.dts |   16 ----
 .../fsl-imx8qxp-mek-lvds0-lvds1-it6263.dts         |   17 ----
 .../dts/freescale/fsl-imx8qxp-mek-lvds1-it6263.dts |   16 ----
 5 files changed, 148 deletions(-)
 delete mode 100644 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-adv7535.dts
 delete mode 100644 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-lvds0-it6263.dts
 delete mode 100644 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-lvds0-lvds1-it6263.dts
 delete mode 100644 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-lvds1-it6263.dts

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index fbfb8c6..277c1a5 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -42,10 +42,6 @@ dtb-$(CONFIG_ARCH_FSL_IMX8QXP) += fsl-imx8qxp-lpddr4-arm2.dtb \
 				  fsl-imx8qxp-mek.dtb \
 				  fsl-imx8qxp-mek-dom0.dtb \
 				  fsl-imx8qxp-mek-enet2.dtb \
-				  fsl-imx8qxp-mek-lvds0-it6263.dtb \
-				  fsl-imx8qxp-mek-lvds1-it6263.dtb \
-				  fsl-imx8qxp-mek-lvds0-lvds1-it6263.dtb \
-				  fsl-imx8qxp-mek-dsi-adv7535.dtb \
 				  fsl-imx8qxp-mek-dsi-rm67191.dtb \
 				  fsl-imx8qxp-lpddr4-arm2-enet2.dtb \
 				  fsl-imx8qxp-lpddr4-arm2-gpmi-nand.dtb \
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-adv7535.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-adv7535.dts
deleted file mode 100644
index 06fe3a2..0000000
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-dsi-adv7535.dts
+++ /dev/null
@@ -1,95 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 2
- * of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- */
-
-#include "fsl-imx8qxp-mek.dts"
-
-&i2c0_mipi_lvds0 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_i2c0_mipi_lvds0>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	adv_bridge1: adv7535@3d {
-		compatible = "adi,adv7535", "adi,adv7533";
-		reg = <0x3d>;
-		adi,dsi-lanes = <4>;
-		status = "okay";
-
-		port {
-			adv7535_1_in: endpoint {
-				remote-endpoint = <&mipi_dsi_bridge1_out>;
-			};
-		};
-	};
-};
-
-&mipi_dsi_phy1 {
-	status = "okay";
-};
-
-&mipi_dsi1 {
-	status = "okay";
-};
-
-&mipi_dsi_bridge1 {
-	status = "okay";
-
-	port@1 {
-		mipi_dsi_bridge1_out: endpoint {
-			remote-endpoint = <&adv7535_1_in>;
-		};
-	};
-};
-
-&i2c0_mipi_lvds1 {
-	#address-cells = <1>;
-	#size-cells = <0>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_i2c0_mipi_lvds1>;
-	clock-frequency = <100000>;
-	status = "okay";
-
-	adv_bridge2: adv7535@3d {
-		compatible = "adi,adv7535", "adi,adv7533";
-		reg = <0x3d>;
-		adi,dsi-lanes = <4>;
-		status = "okay";
-
-		port {
-			adv7535_2_in: endpoint {
-				remote-endpoint = <&mipi_dsi_bridge2_out>;
-			};
-		};
-	};
-};
-
-&mipi_dsi_phy2 {
-	status = "okay";
-};
-
-&mipi_dsi2 {
-	status = "okay";
-};
-
-&mipi_dsi_bridge2 {
-	status = "okay";
-
-	port@1 {
-		mipi_dsi_bridge2_out: endpoint {
-			remote-endpoint = <&adv7535_2_in>;
-		};
-	};
-};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-lvds0-it6263.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-lvds0-it6263.dts
deleted file mode 100644
index 5a7c61b..0000000
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-lvds0-it6263.dts
+++ /dev/null
@@ -1,16 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 2
- * of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- */
-
-#include "fsl-imx8qxp-mek.dts"
-#include "fsl-imx8qxp-mek-lvds0-it6263.dtsi"
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-lvds0-lvds1-it6263.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-lvds0-lvds1-it6263.dts
deleted file mode 100644
index 1391ad7..0000000
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-lvds0-lvds1-it6263.dts
+++ /dev/null
@@ -1,17 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 2
- * of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- */
-
-#include "fsl-imx8qxp-mek.dts"
-#include "fsl-imx8qxp-mek-lvds0-it6263.dtsi"
-#include "fsl-imx8qxp-mek-lvds1-it6263.dtsi"
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-lvds1-it6263.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-lvds1-it6263.dts
deleted file mode 100644
index c14a6a1..0000000
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek-lvds1-it6263.dts
+++ /dev/null
@@ -1,16 +0,0 @@
-/*
- * Copyright 2017 NXP
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 2
- * of the License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- */
-
-#include "fsl-imx8qxp-mek.dts"
-#include "fsl-imx8qxp-mek-lvds1-it6263.dtsi"
-- 
1.7.9.5

